socs/agents/scpi_psu/agent.py

```diff
@@ -145,6 +226,26 @@ def set_current(self, session, params=None):
 
         return True, 'Set channel {} current to {}'.format(params['channel'], params['current'])
 
+    @ocs_agent.param('channel', type=int, choices=[1, 2, 3])
+    def get_output(self, session, params=None):
+        """get_output(channel)
+
+        **Task** - Check if channel ouput is enabled or disabled.
+
+        Parameters:
+            channel (int):
```

Comment:
This looks like it got dropped, but let's just use the same description as the other added methods:
```
channel (int): Channel number (1, 2, or 3).
```

Reply:
fixed